features/supplier/brief_response.feature

```diff
@@ -147,3 +147,94 @@ Scenario: Supplier applies for a user-research-participants brief
       | Liking sugar lumps                 |                      |
       | Being good at jumping over fences  | No jump is too high. |
       | Saying "Neigh"                     | NEIGH                |
+
+Scenario: Previous page links are used during response flow
+  Given that supplier has filled in their application but not submitted it
+    And I have a live digital-specialists brief
+    And that supplier has a completed brief response
```

Comment:
Should it not be:
```
Given I have a live digital-specialists brief
And that supplier has filled in their application but not submitted it
```
So the brief is created before the application is filled in for it

The 'that supplier has a completed brief response' line also being redundant as 'that supplier has filled in their application but not submitted it' covering this?